
using SwinUNETR for training.
  0%|          | 0/686 [00:00<?, ?it/s]
  0%|          | 0/686 [00:04<?, ?it/s]
Traceback (most recent call last):
  File "/mnt/d/PROJECTION/TianChi/STS-2D/train.py", line 193, in <module>
    train_loss = train_step(train_loader, model, criterion, optimizer, CFG.device, epoch)
  File "/mnt/d/PROJECTION/TianChi/STS-2D/train.py", line 68, in train_step
    outputs = model(image.to(device))
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/mnt/d/PROJECTION/TianChi/STS-2D/models.py", line 70, in forward
    x = self.model(x)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/monai/networks/nets/swin_unetr.py", line 297, in forward
    hidden_states_out = self.swinViT(x_in, self.normalize)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/monai/networks/nets/swin_unetr.py", line 1013, in forward
    x2 = self.layers2[0](x1.contiguous())
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/monai/networks/nets/swin_unetr.py", line 890, in forward
    x = blk(x, attn_mask)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/monai/networks/nets/swin_unetr.py", line 675, in forward
    x = x + self.forward_part2(x)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/monai/networks/nets/swin_unetr.py", line 629, in forward_part2
    return self.drop_path(self.mlp(self.norm2(x)))
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/monai/networks/blocks/mlp.py", line 69, in forward
    x = self.fn(self.linear1(x))
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/zy/anaconda3/envs/kaggle_torch/lib/python3.9/site-packages/torch/nn/modules/linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
RuntimeError: CUDA error: unknown error
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.